packages/medusa/src/api/routes/store/customers/reset-password-token.js

```diff
@@ -0,0 +1,25 @@
+import { MedusaError, Validator } from "medusa-core-utils"
+
+export default async (req, res) => {
+  const schema = Validator.object().keys({
+    email: Validator.string()
+      .email()
+      .required(),
+  })
+
+  const { value, error } = schema.validate(req.body)
+  if (error) {
+    throw new MedusaError(MedusaError.Types.INVALID_DATA, error.details)
+  }
+
+  try {
+    const customerService = req.scope.resolve("customerService")
+    const customer = await customerService.retrieveByEmail(value.email)
+
+    await customerService.generateResetPasswordToken(customer._id)
```

Comment:
What happens to the token? As far as I can see, the token is generated but never used.